app/utils/jspsych/functions.js

```diff
@@ -12,6 +13,22 @@ import {
   ExperimentParameters
 } from '../../constants/interfaces';
 
+// create and open an outlet to stream data
+const info = lsl.create_streaminfo(
+  'Dummy',
+  'Marker', // type
+  1, // number of channels
+  10, // sample rate in 1 sec.
+  lsl.channel_format_t.cft_float32,
+  'Dummy Marker Device'
```

Comment:
`Dummy Marker Device` -> `string`

Reply:
I have not found support for strings streaming in node-lsl (see the last comment).